sdk/digitaltwins/azure-digitaltwins-core/README.md

```diff
@@ -28,6 +28,16 @@ authentication as demonstrated below.
 
 ### How to use
 
+#### Import the packages
```

Comment:
I'd prefer including the imports in the sample code for authentication and creating the DT client further below.

Reply:
done.